valkey.conf

```diff
@@ -1248,6 +1248,13 @@ acllog-max-len 128
 # in the system. It's a tradeoff between memory, CPU and latency.
 #
 # active-expire-effort 1
+#
+# This parameter allows Valkey to evict keys earlier. The value represents
+# a percentage of the maxmemory limit, indicating how much memory the Valkey instance reserves
+# without storing data.
+# The default is 0, and the value can be set between 10 and 60.
+#
+# maxmemory-reserved-scale 0
```

Comment:
With the recent update, we are essentially introducing a new "soft" version of "maxmemory". What do you think about the following?

```suggestion
# 
# `maxmemory-soft-scale` defines a "soft" memory threshold as a scale of the `maxmemory` limit.
# When memory usage exceeds this scale, Valkey begins proactive key eviction. However, exceeding this 
# threshold does not immediately reject new write commands; only the hard `maxmemory` limit will do so.
# This scale is specified as a percentage of `maxmemory`, with a valid range between 10 and 60. 
#
# maxmemory-soft-scale 0
```

Reply:
Addressed. Parameter name, variable name, and function name had been changed upon request.